.packit.yaml

```diff
@@ -5,15 +5,18 @@ synced_files:
 upstream_package_name: fmf
 downstream_package_name: fmf
 
+# Epel9 fails to build with dynamic version. Need to create archive with PKG-INFO
+# F37 works with setuptools_scm 7.0
```

Comment:
These comments are not valid anymore, aren't they?

Reply:
F37, no longer relevant. Epel9 unfortunately it is still relevant. Granted, I have never tried setting `SETUPTOOLS_SCM_PRETEND_VERSION`. Shall we test this as well? It would depend on `man` discussion as well, and how https://github.com/teemtee/tmt/pull/3088 evolves

Reply:
ok, let's see if we can get the ball rolling in tmt#3088 next week. If not, let's do whatever works and can be changed later without too much hassle. WDYT?

Reply:
`SETUPTOOLS_SCM_PRETEND_VERSION` still didn't seem to work, so would probably need to keep the workaround. Oh yeah, the comment on F37 is that at least F37 onwards we know it doesn't need the workaround.